src/script/lua_api/l_object.cpp

```diff
@@ -535,14 +542,137 @@ int ObjectRef::l_get_bone_position(lua_State *L)
 		return 0;
 
 	std::string bone = readParam<std::string>(L, 2, "");
+	BoneOverride props = sao->getBoneOverride(bone);
+	push_v3f(L, props.position.vector);
+	v3f euler_rot;
+	props.rotation.next.toEuler(euler_rot);
+	push_v3f(L, euler_rot * core::RADTODEG);
+	return 2;
+}
 
-	v3f position = v3f(0, 0, 0);
-	v3f rotation = v3f(0, 0, 0);
-	sao->getBonePosition(bone, &position, &rotation);
+// set_bone_override(self, bone, override)
+int ObjectRef::l_set_bone_override(lua_State *L)
+{
+	NO_MAP_LOCK_REQUIRED;
+	ObjectRef *ref = checkObject<ObjectRef>(L, 1);
+	ServerActiveObject *sao = getobject(ref);
+	if (sao == NULL)
+		return 0;
 
-	push_v3f(L, position);
-	push_v3f(L, rotation);
-	return 2;
+	std::string bone;
+	if (!lua_isnil(L, 2))
+		bone = readParam<std::string>(L, 2);
```

Comment:
if we're introducing a new api `nil` should probably not silent count for `""`, confusing imo